Sources/LSPLogging/Logging.swift

```diff
@@ -181,11 +187,19 @@ public final class Logger {
     }
   }
 
+  private func logToStderr(_ message: String, level: LogLevel) {
+    let time = self.dateFormatter.string(from: Date())
+    let fullMessage = "[\(time)] \(message)\n"
+    fputs(fullMessage, stderr)
+  }
+
   private func logImpl(_ message: String, level: LogLevel, usedOSLog: Bool) {
 
     if !self.disableNSLog && !usedOSLog {
       // Fallback to NSLog if os_log isn't available.
       NSLog(message)
+    } else {
+      self.logToStderr(message, level: level)
```

Comment:
I just checked: `disableNSLog` is never true except in unit tests.  Should we just rely on NSLog?

Reply:
Hmm we could but won't that duplicate os_logs (os_log from us, NSLog)? If we drop our manual os_log. usage we could but it would be a slight change in behavior (filtering before logging)

Reply:
Ah sorry, I misread how it would work.